fixtures/u2_func_empty.rb.sem.uast

```diff
@@ -0,0 +1,39 @@
+{ '@type': "module",
+   '@role': [Identifier, Module, Statement],
+   file: { '@type': "def",
```

Comment:
Not converted to `uast:FunctionGroup`. Blacklist should be able to detect this.

Reply:
done.